pygmt/src/plot3d.py

```diff
@@ -15,6 +15,8 @@
 
 
 @fmt_docstring
+@deprecate_parameter("columns", "incols", "v0.4.0", remove_version="v0.6.0")
+@deprecate_parameter("sizes", "size", "v0.4.0", remove_version="v0.6.0")
```

Comment:
A test is required to make sure that old parameter `columns` is still supported.

Reply:
Added in b3b78d7